x-pack/solutions/security/plugins/security_solution/docs/rfcs/detection_response/customized_rule_alert_telemetry.md

```diff
@@ -0,0 +1,273 @@
+
+With the implementation of prebuilt rule customization, users are now able to edit almost every field in their prebuilt rules, including functionally dependent fields such as `query`, `filters`, etc. (see full list in the appendix below), which have the potential to change which alerts are generated by the rule. As it exists now, the alert telemetry we gather from users' environments contains limited information about the rule itself, and there is no way to see if the rule is customized at all, let alone which specific fields have been customized. 
+
+We use this alert telemetry in a few different ways, the most notable of which is the TRaDE team's research on how best to tune the rules for threat coverage. We also use it to build the [global threat report](https://www.elastic.co/resources/security/report/global-threat-report) the company releases every year. The current lack of rule customization context in the alert telemetry makes it impossible to distinguish between alerts from unmodified rules and alerts from modified rules, which have the potential to skew this data, leading to less reliable insights gathered. 
+
+For instance, if a rule's query was modified to better suit a user's need to find threats, it could generate vastly different results than the original query would have. With no way to reconcile these changes from a telemetry point of view, the alerts being generated by a different query are looped in with the rest of the unmodified rules' alerts and we are no longer able to trust the collected alerts are being written from the same logic.
+
+### Why this alert telemetry needs to contain information about which fields are customized
+
+The quick solution would be to add the existing `rule_source` field to the alert telemetry [included rule fields](https://github.com/elastic/kibana/blob/0ad59fab8835d67b476ffbdb82850649e70ab4d9/x-pack/solutions/security/plugins/security_solution/server/lib/telemetry/filterlists/prebuilt_rules_alerts.ts), create filters off that, and be done with it. This would unfortunately be too crude of a solution, and would filter out many valuable alerts that have no difference than those generated by an unmodified rules. 
+
+Only customized functional fields - fields that have the ability to modify which alerts are written by a rule - are cause for concern when collecting this alert telemetry data. If a user were to customize a non-functional field such as `tags` or `note`, it would have no influence on the alerts generated by a rule; they're fields that users often change just for workflow management reasons. Changing a functional field, however, like `query`, has ramifications on which events the rule creates alerts from. As stated before, it could generate drastically different results than that of the original, unmodified prebuilt rule which, while probably beneficial to the user, would be entirely unhelpful for our telemetry data that's assuming the original `query` field is being used.
```

Comment:
I think we need a clearer definition of `Functional`. 

```
- Functional Field: ...
- Non-Functional Field: ...
```

It would also be great to specify the process to classify new fields if any ever come.